jmeter-java-dsl/src/main/java/us/abstracta/jmeter/javadsl/core/listeners/InfluxDbBackendListener.java

```diff
@@ -125,6 +127,18 @@ public InfluxDbBackendListener tag(String name, String value) {
     return this;
   }
 
+  /**
+   * Allows specifying a list of percentiles that will be calculated and sent to InfluxDb.
+   * For example "50;95" - will send pct50.0 and pct95.0.
+   *
+   * @param percentiles specifies a list of percentiles separated by ';'.
+   * @return the listener for further configuration or usage.
+   */
+  public InfluxDbBackendListener percentiles(String percentiles) {
```

Comment:
Thank you very much for this contribution!

What do you think of receiving a list of percentiles instead of the user having to know the proper separator for each of them (and potentially mistyping a value providing an invalid number or sequence)? Eg: `public InfluxDbBackendListener percentiles(double... percentiles) {`.

Reply:
Thanks a lot for your suggestions. I think it's a great idea.
I have just done this thing and pushed one more commit. Could you check?